### PR TITLE
feat: Dynamic unlockable tag

### LINKED ISSUE #8649: Pill shape drop notification dynamic
image
- can we make this notification dynamic, so when its not free, it does not say its free - but for example "Generative drop minting now"
- when drop is minted out, it could say, generative drop minted out - see listings

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #8649

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2023-12-18 at 09 55 41@2x

CleanShot 2023-12-18 at 09 57 46@2x

https://github.com/kodadot/nft-gallery/assets/44554284/1d88ac74-72c5-48aa-90de-7febee4dd6d2

## Copilot Summary
copilot:summary

copilot:poem

### CODE REVIEW

**Comment by prury** [comment]
thanks for providing video for minted out drops

File: components/collection/unlockable/utils/useUnlockableTag.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { getDropDetails } from '@/components/drops/useDrops'
+
+const useUnlockableTagTexts = ({
+  isMintedOut,
+  isFree,
```
Comment (daiagi) [inline components/collection/unlockable/utils/useUnlockableTag.ts:5]: to me, creating a non exported composable seems a little pointless
computing the labels directly in `useUnlockableTag` seems more straight forward

File: components/collection/unlockable/UnlockableLandingMobileBanner.vue [hunk 2]
```diff
@@ -7,10 +7,15 @@
         height="42"
         src="/drop/unlockable-pulse-static.svg"
         alt="unlockable icon" />
-      <span>{{ $t('mint.unlockable.mintLive') }}</span>
+      <span>{{ mintStatusText }}</span>
     </div>
-    <nuxt-link class="has-text-weight-bold" :to="DEFAULT_DROP">
-      {{ $t('mint.unlockable.takeMe') }}
+    <nuxt-link class="has-text-weight-bold" :to="to">
```
Comment (daiagi) [inline components/collection/unlockable/UnlockableLandingMobileBanner.vue:12]: Ts warning here on `:to=to` prop

File: components/collection/unlockable/UnlockableLandingTag.vue [hunk 3]
```diff
@@ -9,32 +9,25 @@
...
         alt="unlockable icon" />
-      <span> {{ mintLiveText }} </span>
+      <span> {{ mintStatusText }} </span>
     </div>
     <div class="separator mx-2" />
     <nuxt-link
       class="is-flex is-align-items-center has-text-weight-bold my-2"
-      :to="DEFAULT_DROP">
-      {{ $t('mint.unlockable.takeMe') }}
+      :to="to">
```
Comment (daiagi) [inline components/collection/unlockable/UnlockableLandingTag.vue:17]: ts warnings

**Review by daiagi — CHANGES_REQUESTED** [review]

File: components/collection/unlockable/UnlockableLandingMobileBanner.vue (hunk 2 above)
Comment (hassnian) [reply components/collection/unlockable/UnlockableLandingMobileBanner.vue:12]: CleanShot 2023-12-19 at 17 48 35

File: components/collection/unlockable/UnlockableLandingTag.vue (hunk 3 above)
Comment (hassnian) [reply components/collection/unlockable/UnlockableLandingTag.vue]: no warning on my end
CleanShot 2023-12-19 at 17 47 27

File: components/collection/unlockable/utils/useUnlockableTag.ts (hunk 1 above)
Comment (hassnian) [reply components/collection/unlockable/utils/useUnlockableTag.ts]: move to useUnlockableTag

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 6.78 USD/DOT ~ 7.375 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0xcb462d793e4cd7c8dbfd1f86346055ab64f45c6487ab8d5816a876a0ef75ba70](https://polkadot.subscan.io/extrinsic/0xcb462d793e4cd7c8dbfd1f86346055ab64f45c6487ab8d5816a876a0ef75ba70)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**